### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by GlendaChong**
- Fix headers to match (-ing, pascal case)
- Fix typo, grammar and phrasing
- Enhance introduction section

### CODE REVIEW

**Review by tayruxin — APPROVED** [review]

File: docs/UserGuide.md [hunk 1]
```diff
@@ -7,21 +7,19 @@ title: User Guide
 
 _Master Your Internship Journey: Every Deadline, Every Detail._
 
-LinkMeIn is a **desktop application** built for NUS School Of Computing's Computer Science students 
-to track their internship applications.
+It's the time of the year again! You're scrolling through LinkedIn to discover internship opportunities and tackling programming problems non-stop to prepare for your coding interviews, all in pursuit of securing a summer internship. To increase your chances, you've mass-applied to multiple companies, and now you're struggling to keep track of all your applications.
 
+Well, fret not! LinkMeIn is here to help you! LinkMeIn is a **desktop application** specifically designed for all of you, NUS School Of Computing's Computer Science students, to effectively manage and track your internship applications.
 
 Here is a **quick** overview of your internship journey with LinkMeIn:
```
Comment (papataco14) [inline docs/UserGuide.md:14]: Suggestion: Here's a quick overview of how LinkMeIn transforms your internship application process:

File: docs/UserGuide.md [hunk 2]
```diff
@@ -103,10 +100,10 @@ In this section, you will learn how to set up LinkMeIn in your computer.
 
 5. Open up your command terminal. If you are unsure of how to do so, you can check out [this FAQ](#2-how-do-i-open-up-my-terminal).
 
-6. Navigate to the folder where you have placed the jar file with the `cd` command. 
-   - For example if you have placed the jar file in the **Documents** folder, type `cd Documents` and press Enter.
+6. Navigate to the folder where you have placed the jar file by typing the `cd`, followed by the folder path.
```
Comment (papataco14) [inline docs/UserGuide.md:103]: by typing ~~the~~ `cd`

File: docs/UserGuide.md [hunk 3]
```diff
@@ -131,28 +128,27 @@ The GUI consists of four main components:
 
 ![uiIntro.png](images/ui-intro/uiIntro.png)
 
-Here are what each components stand for:
+Here's what each component stand for:
```
Comment (papataco14) [inline docs/UserGuide.md:131]: component "stands" for

File: docs/UserGuide.md [hunk 4]
```diff
@@ -188,7 +184,8 @@ Let's use the [`add` command](#adding-a-company--add) format as shown below to e
     <img src="images/cli-description/CommandFormat2.png" alt="CommandFormat2" width="650"/>
 </div>
 
-The `add` command is used to initiate the adding a company into LinkMeIn. Every command line should begin with a command word to specify the desired feature.
+The `add` command word is used to initiate the adding a company into LinkMeIn. <br>
```
Comment (papataco14) [inline docs/UserGuide.md:187]: adding "of" a company

File: docs/UserGuide.md [hunk 5]
```diff
@@ -199,21 +196,19 @@ But wait! Notice the `[]` around `pr/PRIORITY`?
...
+Since `PRIORITY` parameter is optional, the following command lines are accepted for the `add` command.
 
 **Example:**
 - `add c/GovTech r/Software Engineer s/PA n/John Doe d/12-12-2023 e/johnDoe@gmail.com p/98765432 pr/medium`
 - `add c/GovTech r/Software Engineer s/PA n/John Doe d/12-12-2023 e/johnDoe@gmail.com p/98765432`
 
 <div markdown="block" class="alert alert-info">
 **:information_source: Notes:**<br>
-* Parameters can be in any order. e.g. if the command specifies c/COMPANY_NAME r/ROLE, r/ROLE c/COMPANY_NAME is also acceptable.
+* Parameters can be in any order. e.g. If the command specifies c/COMPANY_NAME r/ROLE, r/ROLE c/COMPANY_NAME is also acceptable.
```
Comment (papataco14) [inline docs/UserGuide.md:207]: Suggestion: add code blocks for `c/COMPANY_NAME r/ROLE`, and  `r/ROLE c/COMPANY_NAME`, for better clarity

File: docs/UserGuide.md [hunk 6]
```diff
@@ -231,61 +226,56 @@ using quite often. In this tour, you will be learning how and when to:
...
 
 Congratulations! You have successfully added your first company to LinkMeIn. Go ahead and try adding a few more companies on your own!
 Here are some more examples that you can try: 
 - `add c/Oracle r/Data Analyst s/PA n/Marry Lim d/12-11-2023 e/marrylim@example.com p/98452321`
 - `add c/Samsung r/Software Engineer s/PA n/Bob Cheng d/04-03-2023 e/bobcheng@example.com p/88283345 pr/low`
 
-## 2. Viewing company details
+## 2. Viewing Company Details
 
 Now that you had added your first few companies into LinkMeIn, you may want to view the details of a specific company.
```
Comment (papataco14) [inline docs/UserGuide.md:250]: that you "have" added

**Review by papataco14 — CHANGES_REQUESTED** [review]
Some things

**Pull request merged**